src/plugins/intel_cpu/tests/unit/graph/merge_transpose_reorder_test.cpp

```diff
@@ -16,63 +18,97 @@
 #include "common_test_utils/node_builders/constant.hpp"
 
 using namespace ov::intel_cpu;
+using LOOK = Edge::LOOK;
+struct MergeTransposeReorderTestParams {
+    enum Result {IS_OPTIMIZED, NOT_OPTIMIZED};
+    MergeTransposeReorderTestParams(const ov::Shape& testShape,
+                                    ov::element::Type_t testPrec,
+                                    LayoutType firstNodeLayout,
+                                    LOOK firstNodeInplaceDirection,
+                                    LayoutType secondNodeLayout,
+                                    LOOK secondNodeInplaceDirection,
+                                    Result test_result)
```

Comment:
do we really need the constructor while it is possible to use the aggregate initialization?